docs/user-guide/api-mediation/wto-message-on-startup.md

```diff
@@ -0,0 +1,100 @@
+# Configuring initial API Mediation Layer startup message for Syslog
+
+:::info Role: system programmer
+:::
+
+:::note Functionality of this feature is available from Zowe 2.18. 
+:::
+
+Startup of the API Mediation Layer can be configured to present a message in the Syslog that the API Mediation Layer is started and ready. This setup is typically used in combination with Workload Automation tools to manage the lifecycle of Zowe. 
+
+This functionality requires the following changes to the zowe.yaml:
+
+1. Change the property `zowe.sysMessages` by adding the value `- "ZWWEAM001I""`
+    ```
+    zowe:
+      # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+      # You can define any Zowe message portions to be checked for and the message added to the
+      # system log upon its logging, truncated to 126 characters.
+      sysMessages:
+        - "ZWEAM001I"
+    ```
+   This change make sure that message containing `ZWEAM001I` will be presented in the Syslog. 
+   
+   **Example:** 
+
+    `2024-09-30 10:17:53.814 <ZWEAGW1:DiscoveryClient-InstanceInfoReplicator-%d:3335> jb892003 INFO  ((o.z.a.g.c.GatewayHealthIndicator)) ZWEAM001I API Mediation Layer started`
+    
+2. Prepare custom logging configuration.
+   The current default logging implementation starts with information about the current time. This message content is unlike the message id which is typical in z/OS. To change this message behavior, configure the logging configuration. For details about how to configure API Mediation logging, see [Customizing Zowe API Mediation Layer logging](./configuration-logging.md). 
```

Comment:
I don't understand the meaning of this and why this is essential information in the instructions. Can you please modify or explain the reason this information is useful?